### DISCUSSION TITLE
Procedura cambio d'anno 2024/2025

### DISCUSSION THREAD

**Discussion by lucacaputo09** [iisgiua/giuaschool#741] [Richieste di aiuto]
Salve, abbiamo necessità di effettuare il cambio d'anno al nuovo a.s. 2024/2025, è già possibile farlo, c'è un comando che possiamo effettuare dal php admin oppure dobbiamo attendere obbligatoriamente la nuova versione?

**Comment by trinko** [comment]
> Salve, abbiamo necessità di effettuare il cambio d'anno al nuovo a.s. 2024/2025, è già possibile farlo, c'è un comando che possiamo effettuare dal php admin oppure dobbiamo attendere obbligatoriamente la nuova versione?

La procedura di cambio d'anno è in fase di revisione. Consiglio vivamente di attendere il rilascio della nuova versione.
